doc/examples/kubeletconfig.yaml

```diff
@@ -10,6 +10,7 @@ spec:
     cpuManagerPolicy: "static"
     cpuManagerReconcilePeriod: "5s"
     reservedSystemCPUs: "0,1"
+    memoryManagerPolicy: "Static"
```

Comment:
please add to the commit message something like "because otherwise the memory resources are not reported correctly by the kubelet, thus by RTE"

Reply:
done

Reply:
thanks!